zkg

```diff
@@ -946,6 +1131,12 @@ def cmd_purge(manager, args, config, configfile):
             print('Unknown error removing "{}"'.format(name))
             had_failure = True
 
+    try:
+        _dep_pickle_path = os.path.join(manager.scratch_dir, 'deps.pickle')
+        delete_path(_dep_pickle_path)
+    except Exception as exc:
+        print("Error removing dependencies: ", exc)
+
```

Comment:
Seems like this may be leftover or at least still the wrong spot to be doing this (like mentioned in other comments).

Reply:
Fixed.